webclient/src/modules/autocomplete.js

```diff
@@ -24,25 +24,13 @@ navigatum.registerModule(
         const entries = [];
 
         section.entries.forEach((entry) => {
-          // Search uses DC3 and DC1 to mark the beginning/end
-          // of a highlighted sequence:
-          // https://en.wikipedia.org/wiki/C0_and_C1_control_codes#Modified_C0_control_code_sets
-          const name = new Option(entry.name).innerHTML
-            .replaceAll("\x19", "<em>")
-            .replaceAll("\x17", "</em>");
-          const parsedId = new Option(entry.parsed_id).innerHTML
-            .replaceAll("\x19", "<em>")
-            .replaceAll("\x17", "</em>");
-          const subtextBold = new Option(entry.subtext_bold).innerHTML
-            .replaceAll("\x19", "<em>")
-            .replaceAll("\x17", "</em>");
```

Comment:
I have thought about how users might circumvent this and inject stuff. 
Honestly, I don't see a way how anything can be injected via these fields.

This can basically only happen, if someone either gains hold of our backend, TUMs Backend or tries a “Supplychain” attack (someone proposes a PR). I find the first and the last unlikely and the second as a lot of work…

I can see why you designed it this way, but I would say that this would have to be the case for all the other fields as well…

I can revert this If you like, but probably enable it for all the other fields as well, since I don't see the name/… Fields as special… (Correct me If my take on this is wrong)

Reply:
I know what you mean. The data in the response should not contain any malicious scripts, it's more to be on the safe side and to be consequent. Normally vue.js does escaping for us. This is one of the very few cases where we explicitly disable it (using `v-html`, e.g. https://github.com/TUM-Dev/navigatum/blob/main/webclient/src/index.html#L131)

Reply:
Ah... 😲 
When we only do that for these three fields, this makes sense...